hello-tribuo/pom.xml

```diff
@@ -0,0 +1,175 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.gluonhq.samples</groupId>
+        <artifactId>samples</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>com.gluonhq.samples</groupId>
+    <artifactId>hellotribuo</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+
+    <name>Hello Tribuo</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.release>11</maven.compiler.release>
+        <javafx.version>15</javafx.version>
+        <javafx.plugin.version>0.0.4</javafx.plugin.version>
+        <charm.version>6.0.5</charm.version>
+        <attach.version>4.0.8</attach.version>
+        <main.class>com.gluonhq.samples.sample.Main</main.class>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.tribuo</groupId>
+            <artifactId>tribuo-all</artifactId>
+            <version>4.0.1</version>
+            <type>pom</type>
+        </dependency>
+        <dependency>
```

Comment:
I know these are generated from Gluon Start, but `javafx-base` and `javafx-graphics` are not needed, they are transitive dependencies from `javafx-controls`